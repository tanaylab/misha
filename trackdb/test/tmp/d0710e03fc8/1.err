bash: module: line 1: syntax error: unexpected end of file
bash: error importing function definition for `BASH_FUNC_module'
bash: ml: line 1: syntax error: unexpected end of file
bash: error importing function definition for `BASH_FUNC_ml'
Loading required package: sc5mc
Loading required package: gpatterns

Attaching package: ‘gpatterns’

The following objects are masked _by_ ‘.GlobalEnv’:

    fread, qqv

Loading required package: bindrcpp
Joining, by = "index1"
Joining, by = "index2"
Joining, by = c("index1", "index2", "illumina_index", "lib")
0.00reads [00:00, ?reads/s]64.5Kreads [00:00, 645Kreads/s]128Kreads [00:00, 641Kreads/s] 191Kreads [00:00, 640Kreads/s]254Kreads [00:00, 636Kreads/s]315Kreads [00:00, 627Kreads/s]380Kreads [00:00, 633Kreads/s]443Kreads [00:00, 634Kreads/s]504Kreads [00:00, 626Kreads/s]566Kreads [00:00, 623Kreads/s]628Kreads [00:01, 622Kreads/s]694Kreads [00:01, 634Kreads/s]769Kreads [00:01, 665Kreads/s]846Kreads [00:01, 692Kreads/s]920Kreads [00:01, 707Kreads/s]997Kreads [00:01, 724Kreads/s]
0.00reads [00:00, ?reads/s]39.3Kreads [00:00, 393Kreads/s]88.2Kreads [00:00, 417Kreads/s]138Kreads [00:00, 438Kreads/s] 188Kreads [00:00, 454Kreads/s]237Kreads [00:00, 466Kreads/s]312Kreads [00:00, 524Kreads/s]389Kreads [00:00, 580Kreads/s]468Kreads [00:00, 630Kreads/s]543Kreads [00:00, 663Kreads/s]618Kreads [00:01, 688Kreads/s]695Kreads [00:01, 709Kreads/s]771Kreads [00:01, 724Kreads/s]848Kreads [00:01, 738Kreads/s]924Kreads [00:01, 742Kreads/s]998Kreads [00:01, 743Kreads/s]
0.00reads [00:00, ?reads/s]38.0Kreads [00:00, 380Kreads/s]80.3Kreads [00:00, 392Kreads/s]120Kreads [00:00, 392Kreads/s] 161Kreads [00:00, 399Kreads/s]205Kreads [00:00, 409Kreads/s]246Kreads [00:00, 411Kreads/s]288Kreads [00:00, 412Kreads/s]330Kreads [00:00, 416Kreads/s]374Kreads [00:00, 422Kreads/s]416Kreads [00:01, 422Kreads/s]459Kreads [00:01, 423Kreads/s]500Kreads [00:01, 421Kreads/s]543Kreads [00:01, 422Kreads/s]584Kreads [00:01, 421Kreads/s]626Kreads [00:01, 416Kreads/s]669Kreads [00:01, 420Kreads/s]712Kreads [00:01, 421Kreads/s]754Kreads [00:01, 421Kreads/s]796Kreads [00:01, 417Kreads/s]839Kreads [00:02, 420Kreads/s]882Kreads [00:02, 423Kreads/s]925Kreads [00:02, 424Kreads/s]967Kreads [00:02, 424Kreads/s]
0.00reads [00:00, ?reads/s]44.8Kreads [00:00, 448Kreads/s]88.9Kreads [00:00, 446Kreads/s]134Kreads [00:00, 446Kreads/s] 179Kreads [00:00, 448Kreads/s]218Kreads [00:00, 431Kreads/s]263Kreads [00:00, 436Kreads/s]309Kreads [00:00, 444Kreads/s]357Kreads [00:00, 453Kreads/s]407Kreads [00:00, 466Kreads/s]456Kreads [00:01, 472Kreads/s]505Kreads [00:01, 478Kreads/s]552Kreads [00:01, 431Kreads/s]596Kreads [00:01, 410Kreads/s]643Kreads [00:01, 427Kreads/s]712Kreads [00:01, 482Kreads/s]786Kreads [00:01, 538Kreads/s]859Kreads [00:01, 580Kreads/s]933Kreads [00:01, 620Kreads/s]
0.00reads [00:00, ?reads/s]49.6Kreads [00:00, 496Kreads/s]96.3Kreads [00:00, 487Kreads/s]145Kreads [00:00, 486Kreads/s] 193Kreads [00:00, 484Kreads/s]246Kreads [00:00, 497Kreads/s]302Kreads [00:00, 516Kreads/s]360Kreads [00:00, 533Kreads/s]418Kreads [00:00, 547Kreads/s]476Kreads [00:00, 557Kreads/s]532Kreads [00:01, 556Kreads/s]586Kreads [00:01, 551Kreads/s]645Kreads [00:01, 562Kreads/s]704Kreads [00:01, 569Kreads/s]763Kreads [00:01, 575Kreads/s]821Kreads [00:01, 579Kreads/s]879Kreads [00:01, 572Kreads/s]938Kreads [00:01, 577Kreads/s]996Kreads [00:01, 571Kreads/s]
0.00reads [00:00, ?reads/s]38.6Kreads [00:00, 386Kreads/s]97.3Kreads [00:00, 430Kreads/s]157Kreads [00:00, 469Kreads/s] 215Kreads [00:00, 499Kreads/s]273Kreads [00:00, 521Kreads/s]332Kreads [00:00, 538Kreads/s]390Kreads [00:00, 551Kreads/s]449Kreads [00:00, 561Kreads/s]506Kreads [00:00, 565Kreads/s]566Kreads [00:01, 574Kreads/s]624Kreads [00:01, 576Kreads/s]681Kreads [00:01, 565Kreads/s]739Kreads [00:01, 570Kreads/s]797Kreads [00:01, 573Kreads/s]855Kreads [00:01, 577Kreads/s]913Kreads [00:01, 578Kreads/s]971Kreads [00:01, 573Kreads/s]
0.00reads [00:00, ?reads/s]42.5Kreads [00:00, 425Kreads/s]61.0Kreads [00:00, 304Kreads/s]75.4Kreads [00:00, 189Kreads/s]95.5Kreads [00:00, 192Kreads/s]124Kreads [00:00, 213Kreads/s] 164Kreads [00:00, 248Kreads/s]189Kreads [00:00, 231Kreads/s]213Kreads [00:00, 228Kreads/s]236Kreads [00:01, 215Kreads/s]258Kreads [00:01, 187Kreads/s]278Kreads [00:01, 182Kreads/s]297Kreads [00:01, 166Kreads/s]318Kreads [00:01, 179Kreads/s]352Kreads [00:01, 208Kreads/s]395Kreads [00:01, 246Kreads/s]435Kreads [00:01, 278Kreads/s]476Kreads [00:01, 309Kreads/s]511Kreads [00:02, 233Kreads/s]540Kreads [00:02, 204Kreads/s]565Kreads [00:02, 214Kreads/s]610Kreads [00:02, 254Kreads/s]668Kreads [00:02, 305Kreads/s]725Kreads [00:02, 355Kreads/s]783Kreads [00:02, 402Kreads/s]841Kreads [00:02, 442Kreads/s]898Kreads [00:03, 474Kreads/s]956Kreads [00:03, 501Kreads/s]
0.00reads [00:00, ?reads/s]38.0Kreads [00:00, 380Kreads/s]96.2Kreads [00:00, 424Kreads/s]154Kreads [00:00, 461Kreads/s] 211Kreads [00:00, 488Kreads/s]267Kreads [00:00, 508Kreads/s]325Kreads [00:00, 529Kreads/s]383Kreads [00:00, 544Kreads/s]439Kreads [00:00, 547Kreads/s]497Kreads [00:00, 556Kreads/s]553Kreads [00:01, 557Kreads/s]614Kreads [00:01, 572Kreads/s]675Kreads [00:01, 583Kreads/s]737Kreads [00:01, 592Kreads/s]798Kreads [00:01, 598Kreads/s]859Kreads [00:01, 602Kreads/s]920Kreads [00:01, 605Kreads/s]981Kreads [00:01, 604Kreads/s]
0.00reads [00:00, ?reads/s]21.4Kreads [00:00, 214Kreads/s]30.8Kreads [00:00, 154Kreads/s]38.1Kreads [00:00, 114Kreads/s]52.1Kreads [00:00, 121Kreads/s]61.3Kreads [00:00, 81.3Kreads/s]71.8Kreads [00:00, 87.0Kreads/s]80.4Kreads [00:01, 38.3Kreads/s]87.0Kreads [00:01, 43.7Kreads/s]96.9Kreads [00:01, 52.2Kreads/s]114Kreads [00:01, 65.8Kreads/s] 133Kreads [00:01, 82.0Kreads/s]157Kreads [00:01, 102Kreads/s] 173Kreads [00:01, 107Kreads/s]188Kreads [00:01, 109Kreads/s]212Kreads [00:02, 131Kreads/s]229Kreads [00:02, 122Kreads/s]264Kreads [00:02, 151Kreads/s]285Kreads [00:02, 157Kreads/s]324Kreads [00:02, 191Kreads/s]381Kreads [00:02, 238Kreads/s]438Kreads [00:02, 289Kreads/s]495Kreads [00:02, 339Kreads/s]550Kreads [00:02, 384Kreads/s]606Kreads [00:03, 424Kreads/s]665Kreads [00:03, 463Kreads/s]724Kreads [00:03, 493Kreads/s]784Kreads [00:03, 523Kreads/s]845Kreads [00:03, 545Kreads/s]905Kreads [00:03, 561Kreads/s]966Kreads [00:03, 574Kreads/s]
0.00reads [00:00, ?reads/s]39.1Kreads [00:00, 390Kreads/s]97.3Kreads [00:00, 433Kreads/s]156Kreads [00:00, 469Kreads/s] 215Kreads [00:00, 501Kreads/s]276Kreads [00:00, 529Kreads/s]336Kreads [00:00, 548Kreads/s]396Kreads [00:00, 563Kreads/s]457Kreads [00:00, 575Kreads/s]516Kreads [00:00, 582Kreads/s]577Kreads [00:01, 590Kreads/s]637Kreads [00:01, 591Kreads/s]698Kreads [00:01, 596Kreads/s]758Kreads [00:01, 599Kreads/s]818Kreads [00:01, 600Kreads/s]879Kreads [00:01, 602Kreads/s]940Kreads [00:01, 603Kreads/s]1000Kreads [00:01, 602Kreads/s]
