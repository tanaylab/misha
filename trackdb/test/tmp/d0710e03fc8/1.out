2017-10-11 13:55:37 INFO::doing hamming 1
2017-10-11 13:55:39 INFO::removed 0 ambiguous barcodes
2017-10-11 13:55:39 INFO::doing hamming 1
2017-10-11 13:55:40 INFO::removed 0 ambiguous barcodes
2017-10-11 13:55:41 INFO::writing to /net/mraid14/export/tgdata/users/aviezerl/proj/sc5mC/db/batches/BATCH24/PZM00020/fastq/ZMillu7/raw/split/indexes/ZMillu7_S1_L001_001_indexes.tsv
2017-10-11 13:55:43 INFO::running /net/mraid14/export/tgdata/users/aviezerl/tools/R/3.4/gpatterns/import/map_fastq.py -r1 <(gzip -d -c /net/mraid14/export/tgdata/users/aviezerl/proj/sc5mC/db/batches/BATCH24/PZM00020/fastq/ZMillu7/raw/ZMillu7_S1_L001_R1_001.fastq.gz) -r2 <(gzip -d -c /net/mraid14/export/tgdata/users/aviezerl/proj/sc5mC/db/batches/BATCH24/PZM00020/fastq/ZMillu7/raw/ZMillu7_S1_L001_R2_001.fastq.gz) -i /net/mraid14/export/tgdata/users/aviezerl/proj/sc5mC/db/batches/BATCH24/PZM00020/fastq/ZMillu7/raw/split/indexes/ZMillu7_S1_L001_001_indexes.tsv
